Validated_Partners/BareMetal_Partners.txt

```diff
@@ -2,3 +2,4 @@
  kubecost
  suse
  hashicorp
+ sysdig
```

Comment:
Please remove this. We update this with automation once your changes are approved, tested and merged